shared/constants/profile.js

```diff
@@ -1,4 +1,5 @@
 /* @flow */
+
```

Comment:
It seems like some files have the extra line when the flow declaration uses block comments instead of line comments, so I must have just added it to bring it in line with another file I was looking at.